### ISSUE TITLE
Mods that were previously made precisely for the previous version of Smapi, are now incompatible?

### ISSUE THREAD

**Issue by PiePieDocinho** [ZaneYork/SMAPI-Android-Installer#304]
I'm trying to download the same old mods that I used before the update, but the Stardew valley Expanded that I downloaded just for the previous update doesn't work anymore.

So can I just download the available PC version that will work? And as for the VirtualKeyboard, which I assume is the little window that was available in the upper left corner, which served to monitor the progress of mods and use the CJB item menu, now it doesn't appear anymore. Should I download it? If yes, where?

**Comment by Skishe** [comment]
Its an alpha, don't expect it would work perfectly. And frankly, you don't need virtual keyboard just yet as no mods apart from CP works.

**Comment by PiePieDocinho** [comment]
> Its an alpha, don't expect it would work perfectly. And frankly, you don't need virtual keyboard just yet as no mods apart from CP works.

Yeah!! Honestly I just wanted to know! No need to sound so rude 😃✋

Thank you for replying!!

**Comment by Shockah** [comment]
Yeah, the fact that CP loads, doesn't mean it works properly.
